config/rubocop.yml

```diff
@@ -204,9 +204,15 @@ Style/TrailingBlankLines:
 Style/TrailingCommaInArguments:
   Enabled: true
 
-Style/TrailingWhitespace:
+Layout/TrailingWhitespace:
   Enabled: true
 
 Lint/EndAlignment:
-  AlignWith: variable
+  EnforcedStyleAlignWith: variable
   Enabled: true
+
+Style/SingleLineBlockParams:
+  Enabled: true
+
+Style/SymbolArray:
+  Enabled: false
```

Comment:
@thilonel 
All from Changelog, 2 first renamed yes.
2 news changed their default. That's why i need to adapted it to keep exact same rules that previous